ingestion_tools/scripts/importers/base_importer.py

```diff
@@ -126,6 +133,19 @@ def get_default_config(cls) -> list[dict] | None:
         """
         return None
 
+    @classmethod
+    def get_name_and_path(cls, metadata: dict, name: str, path: str, results: dict[str, str]) -> [str, str, dict]:
+        """
+        Returns the name, path and a dictionary of name and paths for the importer. This method is used to override the
+        name and path for the importer when the destination metadata is provided.
+        :param metadata: the metadata associated to the relevant importer entity
+        :param name: the name identified for the importer entity
+        :param path: the path for the importer entity
+        :param results: a dict of the filename and path for the importer entity
```

Comment:
this `results` parameter is unused so far - why do we need it?

Reply:
This will be used in cases like Alignment, where we might want to merge the data from the finder with data from the metadata.